lib/query_builder/schema.ex

```diff
@@ -33,6 +33,7 @@ defmodule EctoShorts.QueryBuilder.Schema do
         end
 
       filter_field in schema.__schema__(:associations) ->
+        %{queryable: relational_schema} = schema.__schema__(:association, filter_field)
```

Comment:
what would be the best control structure for pattern matching this? What should be returned if pattern match fails?

Reply:
What would cause it to fail? Embedded schemas?, if there's no reason then you can leave it without a control structure imo

Reply:
This returns nil if the association doesn't exist which result in a very confusing error @hlista, we should make some sort of guard to throw a proper error explaining the situation